doc/en/user/source/services/wfs/webadmin.rst

```diff
@@ -82,6 +91,30 @@ WFS 2.0.0 request return GML 3.2 as the default format, WFS 1.1.0 requests retur
 
 For each GML type, there is also an "Override GML Attributes" checkbox. Selecting this (checking the checkbox) will cause attributes to be redefined in the application schema.
 
+Override GML 3.2 MIME type
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The default MIME used for GML 3.2 encoded responses is ``application/gml+xml; version=3.2`` which is the MIME type mandated by OGC WFS 2.0 specification. This MIME type is not identified as XML by most common clients like browsers. 
+
+Option :guilabel:`Override MIME Type` allows the selection of the MIME type that should be used for the responses encoded in GML 3.2.  
+
+.. figure:: img/services_WFS_mimetype.png
+
+The available MIME types are: ``application/gml+xml; version=3.2``, ``text/xml; subtype=gml/3.2`` and ``text/xml``. 
+
+Configure XML Entity Expansion limit on WFS XML readers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default WFS XML readers sets Entity Expansion limit to ``100``, but it can be configured via the ``org.geoserver.wfs.xml.entityExpansionLimit`` system property, or :file:`web.xml` init parameter, or Environment variable.
```

Comment:
```suggestion
By default, the WFS XML reader sets Entity Expansion limit to ``100``, but it can be configured via the ``org.geoserver.wfs.xml.entityExpansionLimit`` system property, or using the :file:`web.xml` init parameter, or by Environment variable.
```